kano_settings/system/display.py

```diff
@@ -122,6 +122,13 @@ def set_hdmi_mode(group=None, mode=None):
 
     set_config_value("hdmi_mode", mode)
 
+# flip screen 180
+def set_flip(display_rotate=None):
```

Comment:
@alex5imon I have tried deleting display_rotate from the boot/config.txt and run the settings again, it writes the last line of config.txt every time so no duplicates or errors